hummingbot/connector/exchange_py_base.py

```diff
@@ -535,8 +535,9 @@ async def _execute_order_cancel(self, order: InFlightOrder) -> str:
         except asyncio.CancelledError:
             raise
         except asyncio.TimeoutError:
-            # Binance does not allow cancels with the client/user order id
-            # so log a warning and wait for the creation of the order to complete
+            # It is not possible to cancel orders before an exchange order id has
```

Comment:
I'm new to Hummingbot, but this seems to be a file containing general functionality; therefore, it seems a bit odd to reference a Binance example, inside a general method.

I'll be happy to revert this particular bit, if the comment indeed is meant to refer only to BInance, in this particular case.